### PR TITLE
About page

### PR DESCRIPTION
**Pull request by GDivino**
- [X] Desktop view
- [x] Tablet view
- [x] Mobile view
- [x] Complete photos

Note:
- Changed app.vue for testing purposes

### CODE REVIEW

**Comment by GDivino** [comment]
- [X] Desktop view

- [X] Tablet view

- [X] Mobile view

Notes:
- Changed App.vue for testing purposes
- Changed the pixel units to vw and vh units
- Applied flex box to the code to make it responsive

**Comment by GDivino** [comment]
From screen 1410px - 1025px, the text goes past the image
image

**Comment by GDivino** [comment]
Relevant screenshots
- Desktop view:
image
- Tablet view:
image
- Mobile view:
image

**Review by Znarfois — APPROVED** [review]
Looks great!

**Comment by Znarfois** [comment]
The video is bigger than what we expect when ran. Please check if it is overwritten in another part of the code

File: public/index.html [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" style="max-width: 100%; overflow-x: hidden;">
```
Comment (Znarfois) [inline public/index.html]: Can we move this to CSS?

File: public/index.html [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
     <title><%= htmlWebpackPlugin.options.title %></title>
   </head>
-  <body>
+  <body style="margin: 0;">
```
Comment (Znarfois) [inline public/index.html]: Move this to CSS as well

File: src/components/About.vue [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+        <div class="vid_container">
+            <video width="960px" height="536px" class="about_video">
+                <source src="" type="">
+            </video>
+        </div>
+        <div class="about_startup-summit">
+            <div class="design"></div>
+            <div class="text">
+                <h1>What is Startup Summit 2021?</h1>
+                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.</p>
```
Comment (Znarfois) [inline src/components/About.vue:12]: Bump design team if this is available na

**Review by Znarfois — CHANGES_REQUESTED** [review]
Just small things, other than that looks great! Good job @GDivino

**Review by Znarfois — APPROVED** [review]

**Pull request merged**